charts/keybase-alertmanager/values.yaml

```diff
@@ -69,3 +69,5 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+messageTemplate: ""
```

Comment:
add a command above it describing how it's supposed to be used

Reply:
done